MAINTAINERS.yml

```diff
@@ -228,8 +227,9 @@ CMSIS API layer:
 
 Common arch:
     status: maintained
-    maintainers:
-        - andrewboie
+    collaborators:
```

Comment:
So @nashif should the "arch common" area be set to orphaned?

Reply:
I do not think we should call this orphaned, this is such a bad status to have and gives a very bad impression, we already talked about this and the need to remove it completely.

Reply:
Well if @andrewboie cannot maintain it, we should elevate one collaborator to maintainer level, then. MAybe ask for volunteers?

Reply:
right now I just want to remove andrew and do not worry about setting maintainers and requiring approval and such, thats why I left maintainers out so this change can go in ASAP. that does not mean the area is orphaned, we have many potential maintainers and owners, we can do that in a separate PR.